lib/util.js

```diff
@@ -64,33 +60,33 @@ export function viewport(obj, options) {
  * @api private
  */
 
-export function save(streams) {
+export async function save(streams) {
 	const files = [];
 
-	function end() {
-		return Promise.all(files.map(file => pify(rimraf, Promise)(file)));
+	async function end() {
+		return await* files.map(file => pify(rimraf, Promise)(file));
 	}
 
-	process.on('SIGINT', () => end().then(process.exit));
+	process.on('SIGINT', async () => process.exit(await end()));
 
-	return Promise.all(streams.map(stream => {
-		return pify(mkdirp, Promise)(this.dest()).then(() => {
-			return new Promise((resolve, reject) => {
-				const dest = path.join(this.dest(), stream.filename);
-				const write = fsWriteStreamAtomic(dest);
+	return await* streams.map(async stream => {
+		return new Promise(async (resolve, reject) => {
```

Comment:
Why did you have to wrap this in a Promise constructor? `async stream => {` is already an async function, so it returns a promise. Instead of `reject`, `resolve`, you can just throw and return.